.github/workflows/build-test-distribute-flavor.yml

```
@@ -310,6 +335,23 @@ jobs:
           dind_as_sidecar: 'true'
+          device: ${{ inputs.device }}
+
+      # Run XPU tests sequentially on xpu runners
+      - name: Run XPU tests (sequential)
+        timeout-minutes: ${{ inputs.single_xpu_test_timeout_minutes }}
+        if: ( inputs.device == 'xpu' && inputs.run_single_xpu_tests == true )
+        uses: ./.github/actions/pytest
+        with:
+          image_tag: ${{ steps.calculate-target-tag.outputs.test_image }}
+          pytest_marks: ${{ inputs.single_xpu_test_markers }}
+          framework: ${{ inputs.framework }}
```

actionlint (1.7.12)

[error] input "framework" is not defined in action "Pytest" defined at "./.github/actions/pytest". available inputs are "cpu_limit", "device", "dind_as_sidecar", "dry_run", "hf_token", "image_tag", "parallel_mode", "platform_arch", "pytest_marks", "start_minio", "test_suite_name", "test_type"

(action)